coffee-chats/src/test/java/com/google/step/coffee/JsonServletRequestTest.java

```diff
@@ -0,0 +1,130 @@
+package com.google.step.coffee;
+
+import com.google.appengine.api.datastore.*;
+import org.junit.Test;
+
+import javax.servlet.http.HttpServletRequest;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class JsonServletRequestTest extends TestHelper {
+  @Test
+  public void testGetRequiredParameterExists() throws Exception {
+    HttpServletRequest request = mock(HttpServletRequest.class);
+    when(request.getParameter("hello")).thenReturn("world");
+
+    JsonServletRequest jsonServletRequest = new JsonServletRequest(request);
+    assertThat(jsonServletRequest.getRequiredParameter("hello"), equalTo("world"));
+  }
+
+  @Test(expected = HttpError.class)
```

Comment:
Rather than use this to validate the test, it may be good to validate the reasoning for the error being thrown too?